features/feature-main/src/main/java/com/ownsnoretracker/feature_main/presentation/MainHomeScreen.kt

```diff
@@ -72,6 +72,9 @@ fun PulsingImage(
     Image(
         painter = painterResource(id = R.drawable.moon),
         contentDescription = "CheckIcon",
-        modifier = Modifier.size(imageSize).clip(CircleShape).clickable { onAnimate(true) }
+        modifier = Modifier
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line